LambdatestEcom/UITestFixture.cs

```diff
@@ -16,7 +16,8 @@ public async Task Setup()
             var playwrightDriver = await Playwright.CreateAsync();
             browser = await playwrightDriver.Chromium.LaunchAsync(new BrowserTypeLaunchOptions
             {
-                Headless = ciEnv == "true"
+                Headless = ciEnv == "true",
+                ExecutablePath = @"C:\Users\avkan\AppData\Local\ms-playwright\chromium-1129\chrome-win\chrome.exe"
```

Comment:
this should not be done like this. If you set this path then this solution will work on your PC only.
What was the problem? You specified path to chrome which should work without this line since playwright version is 1.47.0 and it is exactly supports chrome 129